README.md

```diff
@@ -1,51 +1,39 @@
 
-The second step is carrying out the rescue operations in the most optimal manner. It can done by designating specific roles to
-the rescue workers and alloting manpower to each rescue zone. There would be mainly 3 roles:-
+These locations would be rendered in a Google Map Interface in the rescuers' phones. A cluster of locations would comprise a rescue zone. By clicking on a marker in the map interface, information such as the number of devices in the cluster shall be visible. Now rescue operations can be carried out on a zonal basis. In case, there is no internet connectivity at a disaster site, the victims would be located via their phone's **bluetooth**. The range of a bluetooth is approximately 100 metres. Whenever a rescue worker steps within 100 metre radius of a victim, he would be notified of the victim's presence.
 
-* Scout - One who surveys the area and reports the damage and status of victims
-* Medic - One who is responsible for providing first-aid treatment to the victims
-* Lifter - One who removes wreckage and pulls out victims from underneath
+The second step is carrying out the rescue operations in the most optimal manner. There are mainly two kinds of people involved in the relief effort:
 
-There are only 3 roles in the initial draft but the infrastructure of the application would be such that it would allow defining any number of custom roles as per the requirements.
+* Rescuer - One who transfers victims from the disaster site
+* Medic - One who is responsible for providing first-aid treatment to the victims
 
-Now comes the allocation of manpower. The manpower requirements of each zone would be estimated using **machine learning**.
-For Ex:- A rescue zone containing 10 people might require 2 scouts, 5 lifters and 4 medics.
-In case the prediction is a bit off the mark, the rescue worker would have the full accessibility to change the requirements
-of a zone then and there.
+Now comes the allocation of manpower. The manpower requirements of each zone would be estimated using **machine learning**. For Ex:- A rescue zone containing 10 people might require 5 rescuers and 3 medics. In case the prediction is a bit off the mark, the rescue worker would have the full accessibility to change the requirements of a zone then and there.
 
-A user using this application will be able to perform the following tasks:-
+An individual using this application will be able to perform the following tasks:-
 
-* View which zones are assigned to which users in the Map Overlay Mode
+* View the number of rescuers and medics present in a zone
 * Assign himself to a zone and move out to help those in need
 * Dissociate himself from a zone when most probably everybody in the zone has been saved
 * Request reinforcements from the pool of users who are currently un-assigned
-* Assign a task to a user or group of users ( Would require admin privileges to prevent abuse )
```

Comment:
I think this is necessary. We shouldn't leave every decision to individual members. It is entirely possible the rescuers and categorized into teams such as:-
Team Tango - 5 rescuers, 3 medics
Team Charlie - 4 rescuers, 5 medics
One should be able to give commands to a team as a whole